examples/nas/oneshot/spos/multi_trial.py

```diff
@@ -142,9 +143,12 @@ def _main(port):
                                 val_dataloaders=pl.DataLoader(test_dataset, batch_size=64),
                                 max_epochs=2, gpus=1)
 
-    simple_strategy = strategy.Random()
+    simple_strategy = strategy.Random(model_filter=LatencyFilter(100))
 
-    exp = RetiariiExperiment(base_model, trainer, [], simple_strategy)
+    example_inputs = torch.randn(1, 3, 32, 32)
+
+    base_model.eval()
```

Comment:
it is a little strange to config dummy input and "eval" here, let's discuss in the meeting

Reply:
Agree.
I think this input should be obtained from dataloader.